pallets/rmrk-core/src/lib.rs

```diff
@@ -93,8 +107,16 @@ pub mod pallet {
 	#[pallet::storage]
 	#[pallet::getter(fn get_nfts_by_owner)]
 	/// Stores collections info
-	pub type NftsByOwner<T: Config> =
-		StorageMap<_, Twox64Concat, T::AccountId, Vec<(CollectionId, NftId)>>;
+	pub type NftsByOwner<T: Config> = StorageNMap<
```

Comment:
Does this duplicate storage with the Uniques pallet's `Owner` StorageNMap? I thought #91 would remove this from storage.

Reply:
going to leave that change for #91, if we're removing it we'll do so there.